vnc_networks/neuron.py

```diff
@@ -446,8 +366,8 @@ def plot_synapse_distribution(
         plt.get_cmap()
         X, Y, Z = self.get_synapse_distribution(threshold=threshold)
         if ax is None:
-            fig, ax = plt.subplots(1, 1, figsize=params.FIGSIZE, dpi=params.DPI)[1]
-        assert ax is not None  # needed for type hinting
+            _, ax = plt.subplots(1, 1, figsize=params.FIGSIZE, dpi=params.DPI)
+        #assert ax is not None  # needed for type hinting
```

Comment:
At least for me this line is still needed for type hinting to work because the function might return an `Axes` or `None`.

So without this line VSCode complains on the next line `ax=ax` that `ax` could possibly be None but the `scatter_xyz_2d` function expects an `Axes`